Trading/Exchange/binance/binance_exchange.py

```diff
@@ -269,6 +272,27 @@ def parse_leverage(self, fixed, **kwargs):
             fixed[trading_enums.ExchangeConstantsPositionColumns.LEVERAGE.value]
         return parsed
 
+    def parse_funding_rate(self, fixed, from_ticker=False, **kwargs):
+        """
+        Binance last funding time is not provided
+        To obtain the last_funding_time :
+        => timestamp(next_funding_time) - timestamp(BINANCE_DEFAULT_FUNDING_TIME)
+        """
+        if from_ticker:
+            # no funding info in ticker
+            return {}
+        else:
+            funding_dict = super().parse_funding_rate(fixed, from_ticker=from_ticker, **kwargs)
+            funding_next_timestamp = float(
+                funding_dict.get(trading_enums.ExchangeConstantsFundingColumns.NEXT_FUNDING_TIME.value, 0)
+            )
+            # patch LAST_FUNDING_TIME in tentacle
+            funding_dict.update({
+                trading_enums.ExchangeConstantsFundingColumns.LAST_FUNDING_TIME.value:
+                    funding_next_timestamp - self.BINANCE_DEFAULT_FUNDING_TIME
```

Comment:
should we really update with `0-self.BINANCE_DEFAULT_FUNDING_TIME` when next funding time is 0?